stake-pool/cli/src/main.rs

```diff
@@ -55,6 +61,10 @@ type Error = Box<dyn std::error::Error>;
 type CommandResult = Result<Option<Transaction>, Error>;
 
 const STAKE_STATE_LEN: usize = 200;
+const MAX_ACCOUNTS_TO_UPDATE: usize = 10;
+lazy_static! {
+    static ref MIN_STAKE_BALANCE: u64 = sol_to_lamports(1.0);
```

Comment:
This should include the rent exempt amount

Reply:
See my comments above.

Reply:
Resolving

Reply:
So actually, in this case, we should make sure that no withdrawal ever takes more than the rent exempt reserve from a validator stake account.  Only the owner should be allowed to drain the validator stake accounts by first removing them from the stake pool.

Reply:
Ok, so we can just change constant value (1 SOL) to the stake account rent, right?

Reply:
I think it'll be stake account rent + 1 lamport, just like during init, correct?

Reply:
Yes, makes sense